config/settings.yaml.example

```diff
@@ -37,6 +37,11 @@
 #:logging:
 #  :level: debug
 
+# It is also possible to override settings per environment:
+#:logging:
+#  :test:
+#    :pattern: "[%c|%.1l] %m\n"
```

Comment:
Could you please add a pointer in the comment to `logging.yaml` or somewhere in the docs for more info on the syntax so that users can easily find out what they can do here?